pallets/identity/src/claims.rs

```diff
@@ -481,6 +481,19 @@ impl<T: Config> Module<T> {
         // Sender has to be part of CDDProviders
         Self::ensure_authorized_cdd_provider(cdd_did)?;
 
+        let record = <DidRecords<T>>::get(cdd_did);
+        let cost = secondary_keys.iter().try_fold(
+            0usize,
+            |cost, auth| -> Result<usize, DispatchError> {
```

Comment:
Please review these type annotations; some of them are probably redundant, i.e. you can replace them with `_` or remove the suffix `usize` from the literal.